common/php-functions.h

```diff
@@ -12,6 +12,78 @@
 
```

Comment:
Let's remove `#include <cctype>` or `#include <ctype.h>` from everywhere as it shouldn't be used anymore